include/swift/Sema/ConstraintLocatorPathElts.def

```diff
@@ -195,6 +195,10 @@ CUSTOM_LOCATOR_PATH_ELT(ArgumentAttribute)
 /// The result of a chain of member accesses off an UnresolvedMemberExpr
 SIMPLE_LOCATOR_PATH_ELT(UnresolvedMemberChainResult)
 
+/// A reference or expression that results in value of a callable type either @dynamicCallable
```

Comment:
Nit: wrap to 80 columns.

Reply:
- #- - #- - @[lilyball ](https://github.com/github/docs/blob/main/.github/workflows/check-all-english-links.nu